script/GameObjects/Entities/Ship/Thruster.lua

```diff
@@ -52,9 +52,10 @@ end
 
 function Thruster:render(state)
     if state.mode == BlendMode.Additive then
-        if self.parent:getOwner():getControlling() == GameState.player.currentShip and GameState.player.currentCamera == Enums.CameraMode.FirstPerson then
```

Comment:
it´s prob better to use a multiline comment so that the formatter doesn´t destroy it again ;) 

```lua
--[[
--]]
```

Reply:
I actually did at first, and changed it back, because my editor didn't like it. Updated now.